proposals/proposal_buttons_view.py

```diff
@@ -4,20 +4,31 @@
 
 import discord
 from .proposal_modal import ProposalModal
-from .proposal_selects import DeleteProposalSelect, EditProposalSelect, PreviewProposalSelect
+from .proposal_selects import (
+    DeleteProposalSelect,
+    EditProposalSelect,
+    PreviewProposalSelect,
+)
 
 
 class ProposalButtonsView(discord.ui.View):
     def __init__(self, proposals):
         super().__init__()
         self.proposals = proposals
 
-    @discord.ui.button(label="Create", style=discord.ButtonStyle.green)
-    async def create(self, interaction: discord.Interaction, button: discord.ui.Button):
-        modal = ProposalModal(interaction.channel, None)
-        await interaction.response.send_modal(modal)
+    @discord.ui.button(label="Create Proposal", style=discord.ButtonStyle.green)
+    async def create_proposal(
+        self, interaction: discord.Interaction, button: discord.ui.Button
+    ):
+        view = discord.ui.View()
+        view.add_item(CreateGeneralProposalButton(self.proposals))
```

Comment:
I don't have this set up to run locally, can you provide a screenshot of what this looks like?

Reply:
> I don't have this set up to run locally, can you provide a screenshot of what this looks like?

![image](https://github.com/user-attachments/assets/5ca1faf4-81cf-4e84-a791-6fe0cf1cda93)